site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,245 @@
+
+Our old logging system was becoming obsolete and a newer and efficient logging
+system based on fmtlib was designed and implemented by [jroweboy](https://github.com/jroweboy)
+and [daniellimws](https://github.com/daniellimws).
+It got rid of the ugly console window and made many under the hood improvements
+to the logging system.
+With this, identifying errors from crash logs became a whole lot easier and it
+will help us fix issues faster.
+
+## [Citra-qt: Camera Support](https://github.com/citra-emu/citra/pull/3566) by [zhaowenlan1779](https://github.com/zhaowenlan1779)
+
+One of the highly requested features of Citra was support for using system camera
+/ external camera to scan QR codes, etc.
+[Zhaowenlan1779](https://github.com/zhaowenlan1779) took it upon himself to provide
+Citra with this feature. By making use of the available Qt libraries, he worked
+on the implementation for weeks and finally, with collaboration from other contributors,
+he was able to finish the feature. Citra now boasts of its ability to support
+scanning “still images” or using the system camera to function in a way that the
+3DS camera would.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/Camera.jpg"
+    title="System Camera Support" >}}
+
+## [Citra-qt: Translations](https://github.com/citra-emu/citra/pull/3297) by [wwylele](https://github.com/wwylele) and [The Citra Community](https://community.citra-emu.org)
+
+For a long while now, Citra’s UI had been released only in English and had
+absolutely no support for languages other than English. [wwylele](https://github.com/wwylele)
+discussed this at length with other contributors and finally brought UI translations
+to Citra.
+Citra can now be used in many other languages. Note that these translations are
+maintained by our own community members and the translations are hosted on
+[Transifex](https://www.transifex.com/citra/citra/). You too, can submit a translation
+for your language (if it doesn’t exist already), or review existing translations
+to make them perfect.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/Translation.jpeg"
+    title="Citra - Now available in your language" >}}
+
+## Miscellaneous:
+
+Various other contributions to the external dependencies have been done by
+[Lioncash](https://github.com/lioncash), [Merrymage](https://github.com/merrymage)
+and many others. Several services were converted to the new service framework
+and many minor bug fixes, under the hood feature improvements were done by our
+[contributors](https://github.com/citra-emu/citra/graphs/contributors?from=2018-01-01&to=2018-05-19&type=c).
+We believe ***even the smallest contribution is valuable***  and without these
+[contributors](https://github.com/citra-emu/citra/graphs/contributors?from=2018-01-01&to=2018-05-19&type=c),
+these new features wouldn’t have been possible.
+
+<h3 align="center">
+<b><a href="https://github.com/citra-emu/citra/">Contributions are always welcome !</a></b>
```

Comment:
Not sure if this is a bit tacky? I just dunno how I feel about this since the other blog entries don't have it.

Reply:
I believe that having blog posts primarily, attracts new contributors to work on citra. so having this wouldn't hurt.
However, if it is deemed to be unnecessary, then i will get rid of it.